crates/compilers/src/compile/mod.rs

```diff
@@ -1,6 +1,4 @@
 pub mod many;
-
```

Comment:
Is the inconsistent spacing intentional here?

Reply:
removed it